### PR TITLE
graalvm17-ce, graalvm11-ce: 22.0.0.2 -> 22.1.0

### PR DESCRIPTION
**Pull request by kubukoz**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/compilers/graalvm/community-edition/default.nix [hunk 1]
```diff
@@ -12,9 +12,11 @@ let
...
     much easier
+
+    Don't change these values! They will be updated by the update script, see ./update.nix.
   */
-  graalvm11-ce-release-version = "22.0.0.2";
-  graalvm17-ce-release-version = "22.0.0.2";
+  graalvm11-ce-release-version = "22.1.0";
+  graalvm17-ce-release-version = "22.1.0";
   graalvm11-ce-dev-version = "22.2.0-dev-20220415_1945";
   graalvm17-ce-dev-version = "22.2.0-dev-20220415_1945";
```
Comment (thiagokokada) [inline pkgs/development/compilers/graalvm/community-edition/default.nix:21]: You can delete the `graalvmXX-ce-dev-version` likes I think.

**Review by thiagokokada — CHANGES_REQUESTED** [review]
Looks good, just one clean-up.

File: pkgs/development/compilers/graalvm/community-edition/default.nix (hunk 1 above)
Comment (kubukoz) [reply pkgs/development/compilers/graalvm/community-edition/default.nix]: Thanks :)

**Comment by thiagokokada** [comment]
@ofborg build HentaiAtHome babashka clojure-lsp

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 170687` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>HentaiAtHome</li>
    <li>babashka</li>
    <li>clj-kondo</li>
    <li>clojure-lsp</li>
    <li>dapl-native</li>
    <li>dbqn-native</li>
    <li>graalvm11-ce (graalvmCEPackages.graalvm11-ce)</li>
    <li>graalvm17-ce (graalvmCEPackages.graalvm17-ce)</li>
    <li>jet</li>
    <li>neil</li>
    <li>vscode-extensions.betterthantomorrow.calva</li>
    <li>zprint</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]

**Comment by thiagokokada** [comment]
Tested some packages, they all looks to be working fine.

Just waiting until Hydra rebuild some packages, but I would say this PR is good to go.

**Comment by thiagokokada** [comment]
I think `aarch64-darwin` on ofborg is having issues, but since the PR author tested on it I am going to go and merge it.

**Comment by thiagokokada** [comment]
Thanks.

**Pull request merged**